ocpmodels/trainers/base_trainer.py

```diff
@@ -32,7 +33,6 @@
 )
 from ocpmodels.modules.evaluator import Evaluator
 from ocpmodels.modules.normalizer import Normalizer
-from torch.nn.parallel.distributed import DistributedDataParallel
```

Comment:
I never quite understand why these import rearrangements happen; Anuroop's PR just rearranged these a little while back :)

Reply:
I'm running the pre-commit hooks, it's possible @anuroopsriram is just running black? Would be good to be on the same page.